static_src/components/disclaimer.jsx

```diff
@@ -15,20 +15,52 @@ export default class Disclaimer extends React.Component {
 
   render() {
     let flagImg;
+    let dotGovIconImg;
+    let httpsIconImg;
+    const dotGovIcon = require('cloudgov-style/img/icon-dot-gov.svg');
+    const dotGovAlt = 'Dot gov';
+    dotGovIconImg = <img className={ this.styler('usa-banner-icon','usa-media_block-img') } src={ dotGovAlt } src={ dotGovIcon }</img>;
```

Comment:
i would be interested to hear @el-mapache's opinion on the `styler`. i have seen him remove that. wondering if we can do it here in this PR?

Reply:
I think we can remove the styler. I think the main thing it offers over the `classnames` package is that it de-dupes class names by default, something that I haven't seen come up at all in the code base.